src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/FlowController.java

```diff
@@ -204,55 +203,27 @@
         return flowService.pathFlow(flowId);
     }
 
-
     /**
      * Push flows to kilda ... this can be used to get flows into kilda without kilda creating them
      * itself. Kilda won't expect to create them .. it may (and should) validate them at some stage.
-     *
-     * @param externalFlows a list of flows to push to kilda for it to absorb without expectation of creating the flow
-     *        rules
-     * @return list of flow
      */
     @ApiOperation(value = "Push flows without expectation of modifying switches. It can push to switch and validate.",
             response = BatchResults.class)
     @PutMapping(path = "/push")
     @ResponseStatus(HttpStatus.OK)
-    public CompletableFuture<BatchResults> pushFlows(
-            @RequestBody List<FlowInfoData> externalFlows,
-            @ApiParam(value = "default: false. If true, this will propagate rules to the switches.",
-                    required = false)
-            @RequestParam(value = "propagate", required = false) Optional<Boolean> propagate,
-            @ApiParam(value = "default: false. If true, will wait until poll timeout for validation.",
-                    required = false)
-            @RequestParam("verify") Optional<Boolean> verify) {
-
-        Boolean defaultPropagate = false;
-        Boolean defaultVerify = false;
-        return flowService.pushFlows(externalFlows, propagate.orElse(defaultPropagate), verify.orElse(defaultVerify));
+    public CompletableFuture<BatchResults> pushFlows() {
```

Comment:
nit: add `@Deprecated` annotation and `@deprecated` java doc

Reply:
Added.